tests/components/mikrotik/test_config_flow.py

```diff
@@ -124,14 +119,17 @@ def init_config_flow(hass):
 async def test_options(hass):
     """Test updating options."""
     entry = MockConfigEntry(domain=mikrotik.DOMAIN, data=DEMO_CONFIG_ENTRY)
-    flow = init_config_flow(hass)
-    options_flow = flow.async_get_options_flow(entry)
+    entry.add_to_hass(hass)
 
-    result = await options_flow.async_step_init()
+    flow = await hass.config_entries.options.async_create_flow(
```

Comment:
Use `hass.config_entries.options.async_init` directly.

Reply:
It's the same interface as `hass.config_entries.flow.async_init` but for the options flow.

Reply:
done